cloudigrade/api/views.py

```diff
@@ -77,15 +77,13 @@ def list(self, *args, **kwargs):
 
 @method_decorator(transaction.non_atomic_requests, name="dispatch")
 class DailyConcurrentUsageViewSet(viewsets.GenericViewSet, mixins.ListModelMixin):
-    """
-    Generate report of concurrent usage within a time frame.
-
-    This viewset has to be wrapped in a non_atomic_requests decorator. DRF by default
-    runs viewset methods inside of an atomic transaction. But in this case we have to
-    create ConcurrentUsageCalculationTask to schedule jobs even when we raise a
-    ResultsUnavailable 425 exception.
-    """
+    """Generate report of concurrent usage within a time frame."""
```

Comment:
The description field in the openapi schema pulls the whole docstring, so I moved the implementation details out.